tests/kafka-connect-jdbc/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+docker-compose up -d
+
+echo "Wait until kafka-jdbc-connector is started up!"
+
+KAFKA_CONNECTOR_URL="localhost:8083/"
+
+timeout -s TERM 100 bash -c \
+'while [[ "$(curl -s -o /dev/null -L -w ''%{http_code}'' ${0})" != "200" ]];\
```

Comment:
Maybe the curl available already supports these options:

```
       --retry-connrefused
              In addition to the other conditions, consider ECONNREFUSED as a transient error too for --retry. This option is used together with --retry.

              Added in 7.52.0.

       --retry-delay <seconds>
              Make curl sleep this amount of time before each retry when a transfer has failed with a transient error (it changes the default backoff time algorithm between  re‐
              tries). This option is only interesting if --retry is also used. Setting this delay to zero will make curl use the default backoff time.

              If this option is used several times, the last one will be used.

              Added in 7.12.3.


       --retry-max-time <seconds>
              ...

       --retry <num>
```

Reply:
I first used `curl --retry`. It does not cover the case when a zero length response comes from the server and will stop, which happens here when to kafka-connector is starting up. This is why switch to this approach.